scripts/ci/podman-test.sh

```diff
@@ -20,6 +20,7 @@ sed -i 's/#runtime\s*=\s*.*/runtime = "runc"/' /usr/share/containers/containers.
 # Test checkpoint/restore with action script
 echo "action-script /usr/bin/true" | sudo tee /etc/criu/default.conf
 
+cat /proc/mounts
```

Comment:
Usually, I use /proc/self/mountinfo. It contains more info.

Reply:
Thanks! I've updated the patch to use `/proc/self/mountinfo`.